docs/static/breaking-changes.asciidoc

```diff
@@ -1,5 +1,16 @@
 [[breaking-changes]]
-== Breaking Changes
+
+== Breaking Changes Across PQ Versions
+
+If you are upgrading between Logstash versions and use the persistent queue, we strongly recommend draining or deleting
+your persistent queue prior to the upgrade due to data incompatibilities. We are currently working on resolving these issues in our 6.3.0 release, but extra steps will need to be taken until then.
```

Comment:
Present-tense here sounds a bit weird, and will fall out of date quickly; perhaps:

> From our 6.3.0 release forward, Logstash will automatically upgrade compatible persistent queues in-place, but on previous versions extra steps will need to be taken.

Reply:
++ for this use of tense.